### PR TITLE
Add Vercel guide to cookbooks

### LINKED ISSUE #2513: Create Vercel guide
It is important for @meilisearch/devrel-team's current strategy to publish a guide on using Vercel with Meilisearch. Though the (pre-reorg) @meilisearch/docs-team has previously internally the documentation should keep resources focussed on third-party tools to a minimum, the blog does not currently offer enough visibility for the dev rel to meet its goals.

As a temporary solution, we will publish this guide on the documentation, under the Cookbooks section. @guimachiavelli will quickly review the text to ensure it matches the docs website tone and style guides, but @CaroFG will remain this resource's owner.

## Tasks
- [x] Add guide inside `/learn/cookbooks` dir
- [x] Add guide to `/config/sidebar-learn.json` under `Cookbooks`
- [x] Update `.github/CODEOWNERS` assigning @CaroFG as the owner of the document
- [x] Create PR
- [x] Request reviews from @guimachiavelli and one expert (can probably skip the expert if the guide has already been approved by an engineer elsewhere)

### PR DESCRIPTION
**Pull request by CaroFG**
# Pull Request

Not to be merged before the release of the integration

## Related issue
Fixes #2513 

## What does this PR do?
- Adds guide inside /learn/cookbooks dir
- Adds images inside /assets/images/vercel
- Adds guide to /config/sidebar-learn.json under Cookbooks
- Updates .github/CODEOWNERS assigning @CaroFG as the owner of the document

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

File: learn/cookbooks/vercel.mdx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+Once you click on **Create project**, you should see the following message:
+“Your Meilisearch + Vercel integration is one click away from being completed.” and a button to finish the setup. On clicking the button, it should redirect you to the integration page in your Vercel Dashboard.
+
+![Meilisearch Cloud integration page in Vercel's dashboard](https://raw.githubusercontent.com/meilisearch/documentation/main/assets/images/vercel/06.meilisearch-cloud-integration-page-in-vercel-dashboard.png)
+
+Meilisearch creates two default API keys: [`Default Search API Key` and `Default Admin API Key`](/learn/security/master_api_keys#using-default-api-keys-for-authorization). With the `Default Admin API Key`, you can control who can access or create new documents, indexes, or change the configuration. The `Default Search API Key` can only be used to access the [search route](/reference/api/search). It is the one you want to use in your front end.
```
Comment (guimachiavelli) [inline learn/cookbooks/vercel.mdx:47]:
```suggestion
Meilisearch creates two default API keys: [`Default Search API Key` and `Default Admin API Key`](/learn/security/master_api_keys#using-default-api-keys-for-authorization). 

Use the `Default Admin API Key`, to control who can access or create new documents, indexes, and change index settings. Be careful with the admin key and avoid exposing it in public environments.

Use the `Default Search API Key` to access the [search route](/reference/api/search). This is the one you want to use in your front end.
```

File: learn/cookbooks/vercel.mdx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+Both keys are automatically added to Vercel along with the Meilisearch host or URL.
```
Comment (guimachiavelli) [inline learn/cookbooks/vercel.mdx:49]: Is there a difference between host and URL here? If not, I recommend using just one.

File: learn/cookbooks/vercel.mdx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+You can now go back to your project settings and check the new Meilisearch Environment Variables:
```
Comment (guimachiavelli) [inline learn/cookbooks/vercel.mdx:51]: Prefer imperative over "you can [do X]". Using lowercase for environment variables to keep it consistent with the rest of the docs.

```suggestion
Go back to your project settings and check the new Meilisearch environment variables:
```

File: learn/cookbooks/vercel.mdx [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+- `MEILISEARCH_ADMIN_KEY`
+- `MEILISEARCH_URL`
+- `MEILISEARCH_SEARCH_KEY`
+
+![Display the environment variables in the project settings](https://raw.githubusercontent.com/meilisearch/documentation/main/assets/images/vercel/07.project-settings-environment-variables-tab.png)
+
+The master key–which hasn’t been added to Vercel–grants users full control over an instance, you can find it in your project’s overview on your [Meilisearch Cloud dashboard](https://cloud.meilisearch.com/projects/). Read more about [Meilisearch security](https://www.meilisearch.com/docs/learn/security/master_api_keys).
```
Comment (guimachiavelli) [inline learn/cookbooks/vercel.mdx:58]: This paragraph feels disconnected from the what came before and what comes after. If you want to keep it, give it a bit more context so the user knows what they should do with this key.

If the goal information about an important aspect of Meilisearch security without necessarily integrating it into the regular flow of the guide, consider wrapping it in a `<Capsule></Capsule>` component.

```suggestion
The master key–which hasn’t been added to Vercel–grants users full control over an instance. You can find it in your project’s overview on your [Meilisearch Cloud dashboard](https://cloud.meilisearch.com/projects/). Read more about [Meilisearch security](https://www.meilisearch.com/docs/learn/security/master_api_keys).
```

File: learn/cookbooks/vercel.mdx [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+![Meilisearch Cloud dashboard: overview of the "search-app" project](https://raw.githubusercontent.com/meilisearch/documentation/main/assets/images/vercel/08.meilisearch-cloud-dashboard.png)
+
+From your [Meilisearch Cloud dashboard](https://cloud.meilisearch.com/projects/), you can easily index your documents and manage your settings. 
+We also provide a big set of [SDKs and tools](/learn/what_is_meilisearch/sdks), so you can use Meilisearch in your favorite language or framework. Check out our [quick start guide](https://www.meilisearch.com/docs/learn/getting_started/quick_start#add-documents).
+
+You are now ready to [start searching]((/reference/api/search))!
```
Comment (guimachiavelli) [inline learn/cookbooks/vercel.mdx:65]:
```suggestion
Use the [Meilisearch Cloud dashboard](https://cloud.meilisearch.com/projects/), to index  documents and manage your project settings.

Check out the [quick start guide](https://www.meilisearch.com/docs/learn/getting_started/quick_start#add-documents) for a short introduction on how to use Meilisearch. We also provide many [SDKs and tools](/learn/what_is_meilisearch/sdks), so you can use Meilisearch in your favorite language or framework. 

You are now ready to [start searching]((/reference/api/search))!
```

**Review by guimachiavelli — CHANGES_REQUESTED** [review]
I have made a few comments throughout the guide—mostly minor adjustments to bring the text more in line with the rest of the docs (both in tone/style and formatting).

I have one extra recommendation: break down the content after "## Create a Vercel project […]" with extra headings. This will help users navigate the text more easily and give them a way of quickly assess the required steps.

File: config/sidebar-learn.json [hunk 6]
```diff
@@ -335,6 +335,11 @@
 							"source": "learn/cookbooks/http2_ssl.mdx",
 							"label": "Set up HTTP/2 SSL",
 							"slug": "http2_ssl"
+					},
+					{
+						"source": "learn/cookbooks/vercel.mdx",
+						"label": "Integrate Meilisearch Cloud with Vercel",
```
Comment (Strift) [inline config/sidebar-learn.json:341]: Would "Meilisearch Cloud with Vercel" be enough? Shorter looks better.

File: learn/cookbooks/vercel.mdx [hunk 7]
```diff
@@ -0,0 +1,94 @@
+# Integrate Meilisearch Cloud with Vercel
+
+Getting a Meilisearch Cloud instance created and linked to your Vercel projects has never been easier.
```
Comment (Strift) [inline learn/cookbooks/vercel.mdx:3]: I would go for a "This page will teach you how to xxx" type of intro

File: learn/cookbooks/vercel.mdx [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+
+## Introducing our tools
+
+### What is Vercel?
+
+[Vercel](https://vercel.com/) is a cloud platform for building and deploying web applications. It works out of the box with most popular web development tools.
+
+### What is Meilisearch Cloud?
+
+[Meilisearch Cloud](https://www.meilisearch.com/cloud) is a cloud-based SaaS platform, built on top of the Meilisearch open-source search engine.
```
Comment (Strift) [inline learn/cookbooks/vercel.mdx:13]: > cloud based SaaS platform, built on top of meilisearch oss search engine

This is very descriptive. Can we make it more "marketingy", like a few keywords that recaps the main stengthes of the cloud

File: learn/cookbooks/vercel.mdx [hunk 9]
```diff
@@ -0,0 +1,94 @@
...
+### Add the Meilisearch Cloud integration
+
+Go to the project settings tab and click on **Integrations** on the sidebar menu to the left of your screen.
+
+![Selecting the integration tab in the project settings](https://raw.githubusercontent.com/meilisearch/documentation/main/assets/images/vercel/02.project-settings-integrations-tab.png)
+
+Search for the [Meilisearch Cloud integration](https://vercel.com/integrations/meilisearch-cloud) in the search bar. Click on the **Add integration** button.
+
+<Capsule intent="warning">
+Meilisearch Cloud integration is currently in the beta phase. It's not accessible via Vercel's marketplace search at this time.  However, you can conveniently access the integration by visiting the dedicated [Meilisearch Cloud integration page](https://vercel.com/integrations/meilisearch-cloud) directly.
```
Comment (Strift) [inline learn/cookbooks/vercel.mdx:34]: I would make it shorter:

> Meilisearch integration is currently not accessible via Vercel's marketplace search. You can access the integration directly by visiting the [Meilisearch integration page](https://vercel.com/integrations/meilisearch-cloud).

if you want to write "meilisearch cloud integration" you can go for it, but i feel like "meilisearch integration" is more impactful

**Review by Strift — CHANGES_REQUESTED** [review]
Looks good to me, but I wish we'd say "meilisearch integration" instead of "Meilisearch Cloud integration". It sounds more impactful to me.

Not a big deal though :)

File: config/sidebar-learn.json (hunk 6 above)
Comment (CaroFG) [reply config/sidebar-learn.json:341]: I tried to be consistent with the other cookbooks that start with a verb

File: learn/cookbooks/vercel.mdx [hunk 10]
```diff
@@ -0,0 +1,94 @@
+# Integrate Meilisearch Cloud with Vercel
+
+In this guide you will learn how to link a Meilisearch Cloud instance to your Vercel project.
```
Comment (Strift) [inline learn/cookbooks/vercel.mdx:3]: > Meilisearch Cloud

Add link to https://meilisearch.com/cloud with UTM tracking

File: learn/cookbooks/vercel.mdx [hunk 11]
```diff
@@ -0,0 +1,94 @@
...
+
+## Introducing our tools
+
+### What is Vercel?
+
+[Vercel](https://vercel.com/) is a cloud platform for building and deploying web applications. It works out of the box with most popular web development tools.
+
+### What is Meilisearch Cloud?
+
+[Meilisearch Cloud](https://www.meilisearch.com/cloud) is a cloud-based SaaS platform, built on top of the Meilisearch open-source search engine. It offers a managed search service that is scalable, reliable, and designed to meet the needs of all companies
```
Comment (Strift) [inline learn/cookbooks/vercel.mdx:13]: i think you can remove the "is a cloud-based saas platform, built on top of the meilisearch oss engine"

ie. "[Meilisearch Cloud](https://www.meilisearch.com/cloud) offers a managed search service that is scalable, reliable, and designed to meet the needs of all companies"

(missing period at the end of your sentence)

Comment (Strift) [reply learn/cookbooks/vercel.mdx:13]: also update add UTM params!

File: learn/cookbooks/vercel.mdx [hunk 12]
```diff
@@ -0,0 +1,94 @@
...
+Use the `Default Admin API Key`, to control who can access or create new documents, indexes, and change index settings. Be careful with the admin key and avoid exposing it in public environments.
+
+#### Search API key
+
+Use the `Default Search API Key` to access the [search route](/reference/api/search). This is the one you want to use in your front end.
+
+Both keys are automatically added to Vercel along with the Meilisearch URL.
+
+<Capsule intent="tip">
+The master key–which hasn’t been added to Vercel–grants users full control over an instance. You can find it in your project’s overview on your [Meilisearch Cloud dashboard](https://cloud.meilisearch.com/projects/). Read more about [Meilisearch security](https://www.meilisearch.com/docs/learn/security/master_api_keys).
```
Comment (Strift) [inline learn/cookbooks/vercel.mdx:72]: Add UTM params on link to cloud

File: learn/cookbooks/vercel.mdx [hunk 13]
```diff
@@ -0,0 +1,94 @@
...
+- `MEILISEARCH_URL`
+- `MEILISEARCH_SEARCH_KEY`
+
+![Display the environment variables in the project settings](https://raw.githubusercontent.com/meilisearch/documentation/main/assets/images/vercel/07.project-settings-environment-variables-tab.png)
+
+## Take advantage of the Meilisearch Cloud dashboard
+
+![Meilisearch Cloud dashboard: overview of the "search-app" project](https://raw.githubusercontent.com/meilisearch/documentation/main/assets/images/vercel/08.meilisearch-cloud-dashboard.png)
+
+Use the [Meilisearch Cloud dashboard](https://cloud.meilisearch.com/projects/), to index  documents and manage your project settings.
```
Comment (Strift) [inline learn/cookbooks/vercel.mdx:88]: UTM params

File: learn/cookbooks/vercel.mdx [hunk 14]
```diff
@@ -0,0 +1,94 @@
...
+
+## Resources and next steps
+
+Check out the [quick start guide](https://www.meilisearch.com/docs/learn/getting_started/quick_start#add-documents) for a short introduction on how to use Meilisearch. We also provide many [SDKs and tools](/learn/what_is_meilisearch/sdks), so you can use Meilisearch in your favorite language or framework.
```
Comment (Strift) [inline learn/cookbooks/vercel.mdx:92]: why is this link to docs absolute instead of relative?

**Review by Strift — CHANGES_REQUESTED** [review]
Overall good except for 1 paragraph to cut down and UTM params to add on links.

I would go for:
- `utm_campaign=oss`
- `utm_source=docs`
- `utm_medium=vercel-integration`

(FYI @curquiza)

Don't forget to add the line in the [UTM tracking](https://www.notion.so/meilisearch/64b3356d2ce84d87b4fe22feb91f4d82?v=1612b9eee2e34352ab55ea64f45074ba) page in Notion.

File: learn/cookbooks/vercel.mdx (hunk 14 above)
Comment (CaroFG) [reply learn/cookbooks/vercel.mdx:92]: Good catch, it must have slipped

File: learn/cookbooks/vercel.mdx [hunk 15]
```diff
@@ -0,0 +1,94 @@
...
+- `MEILISEARCH_URL`
+- `MEILISEARCH_SEARCH_KEY`
+
+![Display the environment variables in the project settings](https://raw.githubusercontent.com/meilisearch/documentation/main/assets/images/vercel/07.project-settings-environment-variables-tab.png)
+
+## Take advantage of the Meilisearch Cloud dashboard
+
+![Meilisearch Cloud dashboard: overview of the "search-app" project](https://raw.githubusercontent.com/meilisearch/documentation/main/assets/images/vercel/08.meilisearch-cloud-dashboard.png)
+
+Use the [Meilisearch Cloud dashboard](https://cloud.meilisearch.com/projects/?utm_campaign=oss&utm_source=docs&utm_medium=vercel-integration), to index  documents and manage your project settings.
```
Comment (shivaylamba) [inline learn/cookbooks/vercel.mdx:88]: There is an extra spacing after `to index`

**Review by Strift — APPROVED** [review]
LGTM, except for the extra space :)

**Comment by curquiza** [comment]
(Maybe a rebase would fix the broken link)

**Comment by CaroFG** [comment]
> (Maybe a rebase would fix the broken link. I say maybe because it's a tool we just introduced)

Current branch add-vercel-guide is up to date.

**Review by guimachiavelli — APPROVED** [review]

**Pull request merged**